### ISSUE TITLE
Hide columns in filter based export in thymeleaf

### ISSUE THREAD

**Issue by xDarkice** [dandelion/dandelion-datatables#275]
JSP seems to have a property to exclude certain columns in the export using the display attribute. For thymeleaf this option seems to be missing (at least I couldn't find it in the documentation and adding a dt:display="html" didn't exclude the columns).
Is the feature available somewhere and I didn't specifiy the right attributes or is this feature really missing?

**Comment by tduchateau** [comment]
Indeed, the `dt:display` attribute doesn't exist in the Thymeleaf dialect.

Actually it works a bit differently. In Thymeleaf, we assume that every column will be always rendered in HTML and attributes such as `dt:xls` or `dt:pdf` allow you to configure what will be rendered in XLS and PDF exports respectively.

I would instead advise to use controller-based exports (see http://dandelion.github.io/datatables/features/export/controller-based-exports.html) which is the most flexible way of exporting. Just use the HtmlTableBuilder to configure the contents of each cell.
